### ISSUE TITLE
Spotify Web token - Error 400 Bad Request

### ISSUE THREAD

**Issue by guirem** [MoonshineSG/spotify.google#1]
Hi,

Is it still working for you? I get 400 Bad Request errors to get the spotify web token?
It seems Spotify modified their workflow and added some recaptcha technologies...

**Comment by MoonshineSG** [comment]
you are correct. Spotify introduced a google captcha and it broke the script. Until they remove it (or I find another solution) this will not work..

**Comment by MoonshineSG** [comment]
I just realised that the captcha was introduced a while ago but when I wrote and used this script I was never prompted. It looks like they somehow trace the IP that runs scripts and choose add an extra step for login,

**Comment by jared-94** [comment]
Hello,

any news about loging with spotify on a chromecast?
Thanks

**Comment by MoonshineSG** [comment]
i didn’t check lately, but very unlikely this will be possible again. unless spotify decides to open or semi open the platform... i’ll keep an eye on it....

**Comment by jared-94** [comment]
OK I see.
Maybe if one provide the first access token, we may connect to refresh it?

**Comment by MoonshineSG** [comment]
maybe. it’s worth a try...
it would get complicated if you have multiple accounts like i do (wife has separate account) to keep them from expiring even if not in use for a longer time...

**Comment by jared-94** [comment]
OK thank you for your anwser
